larray/core.py

```diff
@@ -3287,48 +3223,18 @@ def any(values, axis=None):
 # commutative modulo float precision errors
 def sum(array, *args, **kwargs):
     """
-    Sum of array elements over a given axis.
+    Sum of array elements.
+
+    Calls :func:`~larray.LArray.sum` method of LArray class if `array`
+    is an LArray object, uses builtin sum function otherwise.
 
     Parameters
     ----------
     array : LArray or iterable
```

Comment:
We either need to document all possible arguments (after having changed the signature of those functions -- see my comment at #41), or not document them at all (which is fine by me). Having only array is a bit weird.